### ISSUE TITLE
Show Native Window crash on Wayland

### ISSUE THREAD

**Issue by sunnyayyl** [wwmm/easyeffects#3542]
### EasyEffects Version

7.2.1

### What package are you using?

Flatpak (Flathub)

### Distribution

Fedora 41 (Silverblue)

### Describe the bug

The Flatpak version of Easy Effect crash when `Show Native Window` button is pressed when using Wayland.
```
[ERR] Can not open display
[ERR] Error initializing plugin wrapper, code: 5
```
The workaround seems to be disabling Wayland socket (in Flatseal) and force Easy Effect to use X11, then it works properly.

**Comment by vchernin** [comment]
Since native plugins always use x11 to my knowledge, this makes sense. The solution would be to enable the x11 socket in the flatpak manifest by default so both the wayland and x11 sockets can be used at the same time. The issue is that flathub and gnome software will complain that easyeffects is using a legacy windowing system. Since this feature isn’t on by default I’m not sure what should be done here.

**Comment by sunnyayyl** [comment]
Ah, I see, I wonder if this is an issue raised in the native's plugin's issue tracker? Or at least something like a feature request? 
(Thanks!)

**Comment by wwmm** [comment]
>Ah, I see, I wonder if this is an issue raised in the native's plugin's issue tracker? Or at least something like a feature request?

I think it is unlikely some of the LV2 plugins we use will ever be ported to wayland. Calf plugins still use gtk2 for example. And it does not seem this is going to change. Allowing X11 to be used in the Flatpak package is probably the only real solution.

**Comment by sunnyayyl** [comment]
Fair enough, X11 is here to stay for a while so it should be fine (and XWayland exist, anyway). Either way, thanks for your work on this app!

**Comment by wwmm** [comment]
So much has changed that is probably better to close this issue
